Source/relay/TourGuide/Items of the Month/2023/Cursed Monkey Paw.ash

```diff
@@ -242,6 +242,46 @@ void IOTMCursedMonkeysPawGenerateResource(ChecklistEntry [int] resource_entries)
                 get_property("8BitScore") < 10000,
             true
         ),
+        new MonkeyWish(
+            $item[none],
+            $effect[Staying Frosty],
+            HTMLGenerateSpanFont("cold damage race", "blue"),
+            !__quest_state["Level 13"].state_boolean["Elemental damage race completed"] && 
+                __quest_state["Level 13"].state_string["Elemental damage race type"] == "cold",
+			true
+        ),
+        new MonkeyWish(
+            $item[none],
+            $effect[Dragged Through the Coals],
+            HTMLGenerateSpanFont("hot damage race", "red"),
+            !__quest_state["Level 13"].state_boolean["Elemental damage race completed"] && 
+                __quest_state["Level 13"].state_string["Elemental damage race type"] == "hot",
+			true
+        ),
+        new MonkeyWish(
+            $item[none],
+            $effect[Bored Stiff],
+            HTMLGenerateSpanFont("spooky damage race", "gray"),
+            !__quest_state["Level 13"].state_boolean["Elemental damage race completed"] && 
+                __quest_state["Level 13"].state_string["Elemental damage race type"] == "spooky",
+			true
+        ),
+        new MonkeyWish(
+            $item[none],
+            $effect[Sewer-Drenched],
+            HTMLGenerateSpanFont("stench damage race", "green"),
+            !__quest_state["Level 13"].state_boolean["Elemental damage race completed"] && 
+                __quest_state["Level 13"].state_string["Elemental damage race type"] == "stench",
+			true
+        ),
+        new MonkeyWish(
```

Comment:
do we want to potentially only show this if the painted-on bikini wish isn't already showing for zepp? kind of weird to show both, since they do the same thing. just add `get_property_int("zeppelinProtestors") > 79` and only one will show at once

Reply:
Added the check.